tests/codegen/ccode/scripts/arrays_slicing.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+from pyccel.decorators import allow_negative_index
+#==============================================================================
+
+def array_view_C_literals():
+
+    from numpy import array
+    z = array([[1, 2, 3], [4, 5, 6]])
+    v = z[1:, 1:] #pylint:disable=unused-variable
+
+def array_view_C_var():
+
+    from numpy import array
+    z = array([[1, 2, 3], [4, 5, 6]])
+    a = 1
+    v = z[a:, 0:2] #pylint:disable=unused-variable
+
+def array_view_C_negative_indexes_literals():
+
+    from numpy import full
+    z = full((2, 5), 2)
+    v = z[-1:, 0:-3] #pylint:disable=unused-variable
+
+@allow_negative_index(y) #pylint:disable=undefined-variable
```

Comment:
Why are you disabling this error? It is a very important error. You can see this by trying to execute this file in pure python mode. It does not run

Reply:
i thought it was intended to let it work on pyccel

Reply:
No. One of the main rules of pyccel is that the file must run in python in order to be an acceptable input to pyccel. Have a look in other files to see how the decorators are used